projects/pathfinder-and-delius/build.gradle.kts

```diff
@@ -35,4 +35,8 @@ configure<ClassPathExtension> {
         "**/entity/**",
         "**/AppKt.class"
     )
+
+    sonarExclusions = listOf(
+        "**/hmpps/**"
+    )
```

Comment:
Not sure if we need this to be honest - we can still approve/merge the PR without it.  This might bite us as the Pathfinder/SOC code diverges.